app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="sign_up">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Kindly increase the length of your forms and submit button to match the wireframe. They should be longer and cover enough space on the page.

  ![image](https://user-images.githubusercontent.com/54848363/225352752-11f28f6d-092a-4a54-b676-89c8bca8085b.png)